SHITv1.sol

```diff
@@ -61,6 +68,34 @@ contract SHITv1 {
         _;
     }
 
+    function payTribute(uint256 _amount) public onlyDuringBusinessHours returns (bool) {
```

Comment:
I suggest that sheeps should be able to pay every god day.

Reply:
what's a god day?

Reply:
I mean every God's day - even weekends are good for charity IMHO.

Reply:
Can't we make all Mondays prayer days?